web/src/api/apiKeys.ts

```diff
@@ -4,6 +4,7 @@ import { createClient, handleError, handleResponse } from "@/api/api.utils";
 interface ApiKey {
   id: string,
   label: string,
+  name: string,
```

Comment:
The field label was added here for this exact use case, but I left it undone on the server side. It is implemented only on the client side. Can you, please, choose one of them (either remove the name and use the label, or remove the label and keep the name one)?